scale/recipe/views.py

```diff
@@ -318,6 +319,33 @@ def list(self, request):
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    def post(self, request):
+        """Queue a recipe and returns the new job information in JSON form
+
+        :param request: the HTTP POST request
+        :type request: :class:`rest_framework.request.Request`
+        :rtype: :class:`rest_framework.response.Response`
+        :returns: the HTTP response to send back to the user
+        """
+        if request.version != 'v6':
+            raise Http404
+        recipe_type_id = rest_util.parse_int(request, 'recipe_type_id')
+        recipe_data = rest_util.parse_dict(request, 'input', {})
```

Comment:
parse recipe_data from request and split into input (v6 data) and recipe configuration (also v6) objects and pass them to methods.